package.json

```diff
@@ -115,6 +115,7 @@
     "ts-node": "9.1.1",
     "typescript": "^3.8.3"
   },
+  "packageManager": "yarn@1.22.19",
```

Comment:
I am using [corepack](https://nodejs.org/api/corepack.html) on my computer, which comes with the latest Yarn by default (v4). Specifying `packageManager` helps `corepack` pick the right version and install deps without errors.